### ISSUE TITLE
Some small problems

### ISSUE THREAD

**Issue by llc0930** [fairyglade/ly#607]
There are some small problems:

1. Custom colors are not applied to the login box.
2. After pressing the restart button, ly behaves as if it is stuck...it is obviously better to exit ly like the old version.
3. The compile time prompts are not very good.

**Comment by AnErrupTion** [comment]
@llc0930 Hello there!
1. Are they applied to the password box though?
2. Does `/sbin/shutdown -r now` work on your system?
3. What do you mean exactly by this?

**Comment by llc0930** [comment]
Regarding question 3, #606 looks good.

**Comment by llc0930** [comment]
After so long, I finally noticed the new setting item `border_fg`.

@AnErrupTion, I thought I'd rephrase what's really problematic at the moment:
1. The prompt for whether numlock is enabled and the text for clock and big clock are only updated while inputting.
2. When restarting or shutting down, because the interface stays in ly, only the input prompts keep flashing, but the screen rendering is not updated, causing the computer to behave as if it has stopped responding. It's obviously better to exit ly like the old version.
3. During the deletion process of ly, only redundant warnings will be printed, and the deleted files will not be printed out like the old version.
```
warn: openrc service not found.
warn: s6 admin service not found.
warn: dinit service not found.
```
4. After the ly installation is completed, all the information that flashed during the installation process will disappear, and the installed files will not be printed out like the old version.

**Comment by AnErrupTion** [comment]
@llc0930 Hey there! I've gone ahead and fixed point 3 & 4, as well as point 1 partially (only the clocks for now), in the master branch. For the build output I can't do better as Zig automatically clears its build output after it finishes, so you'll only see the list of installed/deleted files.
For point 2 though, termbox2 should shutdown when restarting or shutting down via Ly (with F1/F2) which means it should also clear the screen. Doesn't that happen for you?

**Comment by llc0930** [comment]
The installation and removal process looks much better now, but the deleted `data_directory` is not printed during the removal process. Preserving the build output is not that important. If there are no errors, people usually don't notice them.

At least until I relisted the issue above, the screen didn't seem to clear immediately when restarting or shutting down via Ly.

**Comment by llc0930** [comment]
The execution time of `termbox.tb_shutdown()` seems to be too late? The execution time of the old version is before the power management action.

**Comment by AnErrupTion** [comment]
> The execution time of `termbox.tb_shutdown()` seems to be too late? The execution time of this function in the old version was before the actual power management action was performed.

That shouldn't be the case, since when the function returns, it'll *first* execute all deferred statements (which includes `tb_shutdown()`), and *then* it'll execute the return statement (e.g. `execve("/usr/bin/shutdown")`).
